infra/app/adminweb.bicep

```diff
@@ -0,0 +1,58 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+param storageAccountName string = ''
+param formRecognizerName string = ''
+param contentSafetyName string = ''
+param allowedOrigins array = []
+param appServicePlanId string
+param appCommandLine string = 'python -m streamlit run Admin.py --server.port 8000 --server.address 0.0.0.0 --server.enableXsrfProtection false'
+param applicationInsightsName string = ''
+param keyVaultName string = ''
+param azureOpenAIName string = ''
+param azureAISearchName string = ''
+param speechServiceName string = ''
+@secure()
+param appSettings object = {}
+param serviceName string = 'adminweb'
+param useKeyVault bool
+param openAIKeyName string = ''
+param storageAccountKeyName string = ''
+param formRecognizerKeyName string = ''
+param searchKeyName string = ''
+param contentSafetyKeyName string = ''
+param speechKeyName string = ''
+param keyVaultEndpoint string = ''
+param authType string
+
+module adminweb '../core/host/appservice.bicep' = {
```

Comment:
Sorry, I can't reproduce this problem. I guess you may have encountered network problems that caused the failure during the packaging process, because I checked that the [streamlit module](https://github.com/jongio/chat-with-your-data-solution-accelerator/blob/main/code/backend/requirements.txt#L2) is indeed included in the package.

Reply:
Let me try this again from a clean dev container